DependencyInjection/Compiler/XmlMappingDriverPass.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler;
+
+use Doctrine\ORM\Mapping\Driver\SimplifiedXmlDriver;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+use function array_keys;
+use function count;
+use function sprintf;
+
+final class XmlMappingDriverPass implements CompilerPassInterface
```

Comment:
todo: check if we can do this within our extension instead. We are extending Symfonys `AbstractDoctrineExtension` so it should be doable

Reply:
And in any case we should put in a TODO comment to drop this once we bump ORM version

Reply:
and we should tag the class as `@internal` to be able to drop it without BC break.